### ISSUE TITLE
Responsive layout pass

### ISSUE THREAD

**Issue by guahanweb** [jsconf/jsconf.com#12]
Currently, the entire layout is pretty rigid and doesn't work well on smaller devices. 

We need to take a full pass at the current design and layout with mobile view and accessibility in mind. The individual component layouts are already broken out into Pug partials, so moving to grid or flex layout should be pretty easily attainable.
